### PR TITLE
TDBuild update

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

TDBuild update

# Why

Last locale strings update before GA

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-components/src/localization/locales/de-DE/strings.json [hunk 1]
```diff
@@ -24,18 +24,26 @@
...
+    "offLabel": "Aktivieren",
+    "tooltipDisabledContent": "Camera is disabled",
+    "tooltipOnContent": "Kamera ausschalten",
+    "tooltipOffContent": "Kamera einschalten"
   },
   "microphoneButton": {
     "onLabel": "Stumm schalten",
-    "offLabel": "Stummschaltung aufheben"
+    "offLabel": "Stummschaltung aufheben",
+    "tooltipDisabledContent": "Microphone is disabled",
```
Comment (mgamis-msft) [inline packages/react-components/src/localization/locales/de-DE/strings.json:40]: I just noticed they had not translated this in German

File: packages/react-components/src/localization/locales/de-DE/strings.json [hunk 2]
```diff
@@ -24,18 +24,26 @@
...
   "endCallButton": {
-    "label": "Verlassen"
+    "label": "Verlassen",
+    "tooltipContent": "Anruf verlassen"
   },
   "cameraButton": {
     "onLabel": "Deaktivieren",
-    "offLabel": "Aktivieren"
+    "offLabel": "Aktivieren",
+    "tooltipDisabledContent": "Camera is disabled",
```
Comment (prprabhu-ms) [inline packages/react-components/src/localization/locales/de-DE/strings.json:33]: Not translated.

File: packages/react-components/src/localization/locales/de-DE/strings.json [hunk 3]
```diff
@@ -45,14 +53,18 @@
...
     "menuHeader": "In diesem Anruf",
     "participantsListButtonLabel": "{numParticipants} Personen",
     "muteAllButtonLabel": "Alle stumm schalten",
     "copyInviteLinkButtonLabel": "Einladungslink kopieren"
   },
   "screenShareButton": {
     "onLabel": "Beenden",
-    "offLabel": "Teilen"
+    "offLabel": "Teilen",
+    "tooltipDisabledContent": "Screen sharing is disabled",
```
Comment (prprabhu-ms) [inline packages/react-components/src/localization/locales/de-DE/strings.json:65]: Not translated

Comment (prprabhu-ms) [reply packages/react-components/src/localization/locales/de-DE/strings.json:65]: ... to all langauges.

**Review by prprabhu-ms — APPROVED** [review]
3 strings still missing, for all languages.

**Pull request merged**